ui/desktop/app/templates/scopes.hbs

```diff
@@ -48,6 +48,13 @@
         disabled={{unless this.downloadLink 'disabled'}}
         data-test-download-link
       />
+      <Hds::Button
+        type='button'
+        @color='secondary'
+        @route='cluster-url'
+        @text={{t 'unsupported-controller.buttons.change-cluster-url'}}
+        data-test-change-cluster-url
+      />
     </M.Footer>
   </Hds::Modal>
 {{/unless}}
```

Comment:
hmm... clicking the x on the modal does not work for me

Reply:
Yes that is expected. We decided to keep the modal from being dismissed. The only way to "close" it is to shut down the entire desktop client or navigate to the `/cluster-url` route by clicking the new button we added.